bitsandbytes/nn/modules.py

```diff
@@ -282,10 +282,14 @@ def from_prequantized(
         self.compress_statistics = self.quant_state.nested
         self.quant_type = self.quant_state.quant_type
         self.bnb_quantized = True
+
+        # Note: uint8 is the default. TBD: Should this be in QuantState?
+        self.quant_storage = torch.uint8
+
```

Comment:
@Titus-von-Koeller My thought process here is that this should be OK and if we need to support others for `from_pretrained` we could address that later in a PR. For the time being, it's needed because of the change in `to()` below.

Reply:
I pushed the change to `self.quant_storage = data.dtype` which makes more sense here, resolving this.